tool_quality_checks/importer.py

```diff
@@ -6,18 +6,83 @@
 import logging
 from osgeo import ogr
 from os.path import basename
+from OASDGLDatachecker.tool_quality_checks.correct_import_file import (
+    correct_vector_layer,
+)
 
 logger = logging.getLogger(__name__)
 ogr.UseExceptions()
 
 
-def copy2pg_database(settings, in_filepath, layer_name, schema="public"):
+def importer(db, settings):
+    """
+        Loads your input files into the database for checks
+    """
+    # check if relevant parameters are there:
+    if not hasattr(settings, "manhole_layer") or not hasattr(settings, "pipe_layer"):
+        logger.error("One of the input file path is missing, like manhole or pip")
```

Comment:
Bij foutmeldingen betaalt het zich uit om extra correct te zijn. Als de waarde `manhole_layer` heet moet je niet in de foutmelding zeggen dat `manhole` mist.